src/screens/faq/index.js

```diff
@@ -20,42 +21,44 @@ export default () => {
         src={`${process.env.PUBLIC_URL}/images/light-bulb.jpg`}
       />
       <Heading tag='h1'>
-        Frequently Asked Questions
+        {t('faq')}
       </Heading>
       <Heading tag='h2'>
-        What is a Power of Attorney?
+        {t('whatIsPOA')}
       </Heading>
       <Paragraph>
-        A Power of Attorney is a legal document in which a parent authorizes another adult to care for their minor child. By signing a Power of Attorney, the parent does not give up any parental rights.
+        {t('poaIs')}
       </Paragraph>
       <Heading tag='h2'>
-        What is this form used for?
+        {t('WhatIsThisFormUsedFor')}
       </Heading>
       <Paragraph>
-        This form was created for immigrant parents in Tennessee who face the threat of being detained and/or deported. It allows parents to grant a caregiver temporary rights to make decisions for their minor child on their behalf. The Power of Attorney goes into effect only if the parents have been detained or deported. Completion of this form, along with the proper signatures, authorizes the caregiver to enroll the child in school, obtain medical treatment for the child, and provide for the child’s food, housing, and travel.
+        {t('formWasCreated')}
       </Paragraph>
       <Heading tag='h2'>
-        Who should I choose as my caregiver?
+        {t('whoShouldIChoose')}
       </Heading>
       <Paragraph>
-        Your caregiver should be someone you trust. This document will allow the caregiver the right and responsibility to make decisions for your child on your behalf. The person you designate to care for your child should be a U.S. citizen or someone with immigration status who does not also face the threat of detainment or deportation.
+        {t('yourCaregiver')}
       </Paragraph>
       <Heading tag='h2'>
-        How can I terminate the Power of Attorney?
+        {t('howCanITerminate')}
       </Heading>
       <Paragraph>
-        You may terminate the Power of Attorney at any time by putting in writing your desire to do so.
+        {t('youCanTerminate')}
       </Paragraph>
       <Heading tag='h2'>
-        Where can I learn more about Power of Attorney for the care of a minor child?
+        {t('whereCanILearn')}
       </Heading>
       <Paragraph>
-        The law can be found <Anchor
+        {t('lawCanBeFound')} <Anchor
```

Comment:
You could wrap the content of this `Paragraph` with a [`Trans`](https://react.i18next.com/components/trans-component) component to translate the entire string, React components and all.